source/manual/get-started.html.md

```diff
@@ -17,24 +17,278 @@ If you're having trouble with this guide, you can ask your colleagues or the #go
+```
+
+## 3. Set up govuk-docker
 
 We use a Docker environment - [govuk-docker][] - for local development.
 
 👉 [Learn about how we use Docker](/manual/intro-to-docker.html)
 
 👉 [Get setup with govuk-docker][govuk-docker]
 
-## 3. Get SSH access to integration
+[govuk-docker]: https://github.com/alphagov/govuk-docker
 
-👉 [Get SSH access to integration](/manual/get-ssh-access.html)
+## 4. Get SSH access to integration
 
-## 4. Get AWS access
+### Get access
 
-👉 [Setup your AWS account](/manual/set-up-aws-account.html)
+Ask somebody with access to add your SSH username (`firstnamelastname`) to the [user monitoring system][user-reviewer].
+
+[user-reviewer]: https://github.com/alphagov/govuk-user-reviewer
+
+### Create a user to SSH into integration
+
+User accounts in our integration environments are managed in the [govuk-puppet][] repository.
+
+    mac$ mkdir ~/govuk
+    mac$ cd ~/govuk
+    mac$ git clone git@github.com:alphagov/govuk-puppet.git
+
+Now create a user manifest in `~/govuk/govuk-puppet/modules/users/manifests` with your username and the public key you created when you set up your GitHub account above. Your file should use the `firstnamelastname.pp` format.
+
+```
+class users::johnsmith {
+  govuk_user { 'johnsmith':
+    fullname => 'John Smith',
+    email    => 'john.smith.123@digital.cabinet-office.gov.uk',
+    ssh_key  => 'this public key will be a few lines long (copy the output from `more ~/.ssh/alphagov.pub`). It should begin with `ssh-rsa AAA` and end with `== john.smith.123@digital.cabinet-office.gov.uk`. You may need to add the email address to the end of your public key manually.',
+  }
+}
+```
+
+Add the name of your manifest (your username) into the list of `users::usernames` in [`hieradata_aws/integration.yaml`][integration-aws-hiera] for integration and in [`hieradata/integration.yaml`][integration-hiera] for CI.
+
+Create a pull request with these changes. Once it has been [reviewed by a member of the GOV.UK team][merging], you can merge it and it will automatically deploy to the integration environment.
+
+[govuk-puppet]: https://github.com/alphagov/govuk-puppet
+[integration-aws-hiera]: https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/integration.yaml
+[integration-hiera]: https://github.com/alphagov/govuk-puppet/blob/master/hieradata/integration.yaml
+[merging]: /manual/merge-pr.html
+
+### Access remote environments
+
+Your pull request from earlier will hopefully have been merged by now. It's time to test your access to servers via SSH.
+
+> If you're not in the office right now, you'll need to be connected to the GDS office VPN for SSH access to integration.
+
+Test that it works by running:
+
+```bash
+gds govuk connect --environment integration ssh backend
+```
+
+#### Running a console
+Once you have SSH'd into a machine, you can also open a console for a particular application so you can execute commands, for example:
+
+```bash
+govuk_app_console publishing-api
```

Comment:
Make this one of the apps that live on a backend machine, like `signon`

Reply:
Oh, hah. Yes.